src/lib.js

```diff
@@ -28,11 +29,22 @@ const dePseudify = (function () {
              */
             '::?-(?:moz|ms|webkit|o)-[a-z0-9-]+'
         ],
-        // Actual regex is of the format: /([^\\])(:hover|:focus)+/
-        pseudosRegex = new RegExp('([^\\\\])(' + ignoredPseudos.join('|') + ')+');
 
-    return function (selector) {
-        return selector.replace(pseudosRegex, '$1');
+        // Actual regex is of the format: /^(:hover|:focus|...)$/i
+        pseudosRegex = new RegExp('^(' + ignoredPseudos.join('|') + ')$', 'i');
```

Comment:
Isn't this indented too far?

Reply:
I actually didn't change the indentation of that (it's still in the 9th column after 8 spaces), but I can see how it looks weird followed by the new `function` (which is required by the `func-style` in-use by the project; I can't make it another `const` and have the linting pass) and `const` definition after that (which uses the function).

I could give `pseudosRegex` its own `const`, but *then* eslint wants me to dedent the entire `ignoredPseudos` block since it would then by itself, and that's a bigger diff.

I can change the formatting here to whatever you'd like, though, if you don't mind the bigger diff.

Reply:
NVM, git just collapsed the diff at weird spots and made this look wrong. The indentation is perfectly fine.